crates/server/README.md

```diff
@@ -0,0 +1,214 @@
+# Node Server
+
+- [Introduction](#introduction)
+    - [Admin API](#1admin-api)
+    - [JSON rpc](#2json-rpc)
+    - [Websocket](#3websocket)
+- [Admin API endpoints](#admin-api-endpoints)
+    - [Protected Routes](#protected-routes)
+    - [Unprotected Routes](#unprotected-routes)
+- [JSON rpc endpoint](#json-rpc-endpoint)
+- [Websocket endpoints](#websocket-endpoints)
+- [Examples](#examples)
+
+## Introduction
+
+Node Server is a component in node that facilitates node administration and enables communication with the logic of an application (loaded wasm) in participating contexts.
+
+Node Server component is split into 3 parts:
+ 1. [Admin API](https://github.com/calimero-network/core/blob/feat-admin_api_docs/crates/server/src/admin/service.rs)
+ 2. [JSON rpc](https://github.com/calimero-network/core/blob/feat-admin_api_docs/crates/server/src/jsonrpc.rs)
+ 3. [Websocket](https://github.com/calimero-network/core/blob/feat-admin_api_docs/crates/server/src/ws.rs)
+
+### 1. Admin API
+
+The Admin API component of the Node Server exposes API for connection with the node and its functionalities. It is primarily utilized by the Admin Dashboard to query and manage various aspects of the node, including:
+
+ - Identity information
+ - Root keys
+ - Client keys
+ - Installed applications
+ - Started contexts
+
+
+**Data Querying**: The Admin API allows the Admin Dashboard to fetch important data from the node, such as identity details, root and client keys, and information about installed and active applications.
+
+**Application Management**: The API provides functionalities for managing applications and contexts, allowing for the installation and uninstallation of applications and starting contexts.
+
+**Key Management**: Administrators can manage root and client keys through the API, ensuring secure access and control over the node.
+
+**Authentication**: The Admin API facilitates user authentication via selected wallets, currently supporting MetaMask and NEAR networks. Authentication details will be explained in later sections.
+
+**Integration with Web Applications**: The authentication mechanism is also used by web applications designed for applications (loaded wasm) in participating context, ensuring secure and authenticated access.
+
+### 2. JSON rpc
+
+The JSON-RPC component of the Node Server facilitates communication between the clients and the application binaries (contexts) running on node. This allows seamless interaction and data management for applications.
```

Comment:
`application binaries (contexts)` - context is more that just application binary


`communication between the clients and the context. `

Reply:
Updated